python/lsst/ts/watcher/alarm.py

```diff
@@ -329,6 +332,8 @@ async def mute(self, duration, severity, user):
     async def unmute(self):
         """Unmute this alarm."""
         self._cancel_unmute()
+        if self.max_severity == AlarmSeverity.CRITICAL:
```

Comment:
Are you sure using the `max_severity` is really what we need to do? Shouldn't it be `severity` instead? 

For example, if a CSC is in FAULT and we mute the alarm, then put it back I STANDY or DISABLED (like we have been doing with the MTMount recently) there's no need to escalate the alarm.

Reply:
We talked about this on Slack and concluded the following:
* This code has a bug: it should only restart the the escalation timer if the alarm has not been acknowledged. I will fix that.
* At present muting an alarm does not acknowledge it. That should perhaps be changed in a future ticket.
* If an alarm is displayed as unacked and has max severity CRITICAL then it should *always* be escalated (after a suitable interval, and if escalation is configured). Doing anything else would be too confusing.